scss/_variables.scss

```diff
@@ -1069,11 +1069,13 @@ $form-file-button-hover-bg:       var(--#{$prefix}secondary-bg) !default;
 
 // scss-docs-start form-floating-variables
 $form-floating-height:                  add(3.5rem, $input-height-border) !default;
-$form-floating-line-height:             1.25 !default;
+$form-floating-line-height:             1.375 !default;
```

Comment:
I think I would keep the previous value for the line-height so it won't break anything for people and keep the changes very tight to the initial problem.

Reply:
Ah yes my bad, I didn't test on Chrome but on FF and it was causing no issue.